### PR TITLE
[PIN-2282] Getting name using tenant and removed Party-Management

### PR DESCRIPTION
**Pull request by bloodsky**
Using the tenant management service I obtained the name field that previously were needed to be fetched from Selfcare Party Management!

### CODE REVIEW

File: src/main/scala/it/pagopa/interop/agreementprocess/service/util/PDFPayload.scala [hunk 1]
```diff
@@ -15,8 +14,8 @@ final case class PDFPayload(
   today: OffsetDateTime,
   agreementId: UUID,
   eService: String,
-  producer: Institution,
-  consumer: Institution,
+  producer: String,
+  consumer: String,
```
Comment (galales) [inline src/main/scala/it/pagopa/interop/agreementprocess/service/util/PDFPayload.scala]:
```suggestion
  producerName: String,
  consumerName: String,
```

**Review by galales — COMMENTED** [review]
there should be references to party also in 
- `ApplicationConfiguration`
- `Versions`
- `application.conf`
- `application-standalone.conf`
- `application-test.conf`

You can remove also references to party management api key

File: src/main/scala/it/pagopa/interop/agreementprocess/service/util/PDFPayload.scala (hunk 1 above)
Comment (bloodsky) [reply src/main/scala/it/pagopa/interop/agreementprocess/service/util/PDFPayload.scala]: Smart rename! thx 😄

**Review by galales — APPROVED** [review]
🚀

**Review by beetlecrunch — APPROVED** [review]

**Review by TonioGela — APPROVED** [review]

**Comment by TonioGela** [comment]
This may be due to some dependency that has still not been released

```
[error] /home/jenkins/agent/workspace/op-be-agreement-process_PIN-2282/lifecycle/src/test/scala/it/pagopa/interop/agreementprocess/lifecycle/SpecData.scala:34:10: unknown parameter name: name
[error]     name = "test_name"
[error]          ^
[error] /home/jenkins/agent/workspace/op-be-agreement-process_PIN-2282/lifecycle/src/test/scala/it/pagopa/interop/agreementprocess/lifecycle/SpecData.scala:25:30: too many arguments (found 9, expected 8) for method apply: (id: java.util.UUID, selfcareId: Option[String], externalId: it.pagopa.interop.tenantmanagement.client.model.ExternalId, features: Seq[it.pagopa.interop.tenantmanagement.client.model.TenantFeature], attributes: Seq[it.pagopa.interop.tenantmanagement.client.model.TenantAttribute], createdAt: java.time.OffsetDateTime, updatedAt: Option[java.time.OffsetDateTime], mails: Seq[it.pagopa.interop.tenantmanagement.client.model.Mail]): it.pagopa.interop.tenantmanagement.client.model.Tenant in object Tenant
[error]   def tenant: Tenant = Tenant(
[error]                              ^
[error] two errors found
[info] compiling 19 Scala sources to /home/jenkins/agent/workspace/op-be-agreement-process_PIN-2282/generated/target/scala-2.13/classes ...
[warn] 3 feature warnings; re-run with -feature for details
[warn] one warning found
[info] done compiling
[info] compiling 31 Scala sources to /home/jenkins/agent/workspace/op-be-agreement-process_PIN-2282/target/scala-2.13/classes ...
[error] /home/jenkins/agent/workspace/op-be-agreement-process_PIN-2282/src/main/scala/it/pagopa/interop/agreementprocess/service/AgreementContractCreator.scala:143:37: value name is not a member of it.pagopa.interop.tenantmanagement.client.model.Tenant
[error]       producerName = producerTenant.name,
[error]                                     ^
[error] /home/jenkins/agent/workspace/op-be-agreement-process_PIN-2282/src/main/scala/it/pagopa/interop/agreementprocess/service/AgreementContractCreator.scala:144:37: value name is not a member of it.pagopa.interop.tenantmanagement.client.model.Tenant
[error]       consumerName = consumerTenant.name,
[error]                                     ^
[error] two errors found
[error] (lifecycle / Test / compileIncremental) Compilation failed
[error] (Compile / compileIncremental) Compilation failed```

**Comment by TonioGela** [comment]
CI is now green

**Comment by bloodsky** [comment]
Merging right now! Thanks! 😄

**Pull request merged**